### DISCUSSION TITLE
How to run app in android studio guide needed

### DISCUSSION THREAD

**Discussion by ramjohanson** [Akylas/OSS-DocumentScanner#137] [General]
I was going through the project. I am struggling on how to run an after in android studio because app is using C++ & react mixture. I am new to it. please guide me.

**Comment by farfromrefug** [comment]
@ramjohanson The app is using Nativescript which is a cross platform framework. So to run the app you dont need android studio.
To run the app:
* follow the steps on the readme to get 3rd party libraries (opencv and tesseract)
` yarn`
`ns run android --no-hmr`

That should be enough to run build the app. If you need to debug the c++ side then it get a bit different as we need to use android studio with the generated project (on the first `ns run android`)

**Comment by jacoblal** [comment]
I wish, this code can be in native android java. Is there any way to convert entire react code in java?

**Reply by farfromrefug** [reply]
there is no react code ! 
and no it cant be translated to java. and there would be no point as it would not be a cross platform app anymore. and it would take me twice the time to maintain iOS/android

**Reply by jacoblal** [reply]
How can i convert it to java?

**Reply by farfromrefug** [reply]
I Just told you you cant
